records_mover/records/pandas/read_csv_options.py

```diff
@@ -370,9 +372,14 @@ def pandas_read_csv_options(records_format: DelimitedRecordsFormat,
     # Note: A fast-path exists for iso8601-formatted dates.
     #
 
-    # (we don't yet pass in a records schema which would provide
-    # ability to know in advance which columns are datetimes--sounds
-    # like it may be very helpful to do so!)
+    pandas_options['parse_dates'] = [
+        index
+        for index, field
+        in enumerate(records_schema.fields)
+        if field.field_type in ['date', 'time', 'datetime', 'datetimetz']
+    ]
+
+    # TODO: Evaluate below
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker